<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Produto</class>
 <widget class="QDialog" name="Produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Produto</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tipo</string>
    </attribute>
    <widget class="QGroupBox" name="_tipo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>451</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Cadastrar Tipo</string>
     </property>
     <widget class="QPushButton" name="pushButton_inserirtipo">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Novo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tipo">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_apagartipo">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_atualizatipo">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Atualizar</string>
      </property>
     </widget>
     <widget class="QListView" name="listView_tipo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>301</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Descrição:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tamanho</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_tamanho">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>Cadastrar Tamanho</string>
     </property>
     <widget class="QTableView" name="tableView_tamanho">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>431</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_desctamanho">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_seltamanho">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Descrição</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>59</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleção</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_instamanho">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Novo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_del_tamanho">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_tamanho_atualizar">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Atualizar</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Itens</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>621</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Cadastrar Item</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_nomeitem">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>441</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_descricaoitem">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>441</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_novoitem">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Novo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_atualizaritem">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>90</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Atualizar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_apagaritem">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>140</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Apagar
seleção</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_procuraritem">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Procurar</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_itens">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>511</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_iditem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>59</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Descrição</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_procurar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>421</width>
        <height>22</height>
       </rect>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
